discovery-provider/src/queries/queries.py

```diff
@@ -2251,3 +2251,20 @@ def get_previously_private_playlist():
         playlist_ids = [result[0] for result in previously_private_results]
 
     return api_helpers.success_response({ 'ids': playlist_ids })
+
+# Get the users for a given creator node ID
+@bp.route("/creator_node/users/<int:sp_node_id>", methods=("GET",))
+def get_creator_node_users(sp_node_id):
```

Comment:
thoughts on pagination?
also can we pass a flag to return userId only? my guess is we really only need the ID for base case, and maybe wallet

Reply:
isn't 'users/creator_node/<int:sp_node_id>' the current state?

Reply:
nvm - will changee

Reply:
per offline, will return everything

Reply:
added